snap/hooks/configure

```diff
@@ -99,3 +99,10 @@ then
     systemctl restart snap.${SNAP_NAME}.daemon-apiserver.service
     systemctl restart snap.${SNAP_NAME}.daemon-proxy.service
 fi
+
+# Cache Juju image if juju is present and this is an install/refresh
+if [ ! -e ${SNAP_DATA}/.cache-juju-image ]
+then
+   maybe_cache_juju_operator_images >> /tmp/mk8s.log
```

Comment:
Could we place the log in side "$SNAP_DATA" or "$SNAP_COMMON"? It will make our life easier if we decide to go with strict confinement?

Reply:
d'oh, this is left over cruft for when I was testing/debugging. I could log that to a file in DATA/COMMON if wanted though.

Reply:
I am fine with not logging it.